todos/week3/4_classestodo.py

```diff
@@ -1,12 +1,14 @@
 """
 Creating classes for your classmates
 """
-
+# TODO: #############################################################################
+# TODO: Section 1:
 # TODO: Create a class for an Employee, and include basic data
 # TODO: like hours worked, salary, first name, last name, age, and title
 
 
-
+# TODO: ##############################################################################
```

Comment:
The separation of sections should be equal in their lengths. 100 characters is preferred